src/ProjectTemplates/Microsoft.Extensions.AI.Templates/Microsoft.Extensions.AI.Templates.csproj

```diff
@@ -24,6 +24,7 @@
 
   <ItemGroup>
     <Content Include="src\ChatWithCustomData\**\*" Exclude="**\bin\**;**\obj\**;**\node_modules\**;**\*.user;**\*.csproj.in;**\*.out.js;**\*.generated.css;**\package-lock.json" />
+    <None Include="THIRD-PARTY-NOTICES.TXT" Pack="true" PackagePath="." />
```

Comment:
Do we typically include these with .NET templates? Looks like this will be included in the project once created, right?

Reply:
It will be included in the NuGet package but not included as content in the generated template (similar to the `README.md` in the same folder).

Reply:
The same thing is done in, e.g., [Microsoft.AspNetCore.SignalR.Client](https://nuget.info/packages/Microsoft.AspNetCore.SignalR.Client/9.0.2).

Reply:
Ah, OK. Perfect